rambox.metainfo.xml

```diff
@@ -0,0 +1,41 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>app.rambox.ramboxce</id>
```

Comment:
```suggestion
  <id>rambox</id>
```

Reply:
this causes an error: F: No matching appdata for app.rambox.RamboxCE

Reply:
when do you see this error?  sounds like not everything was in a matching state.

Reply:
right before pruning cache:
`% flatpak-builder .build app.rambox.Rambox.yaml --force-clean --install --user        
Emptying app dir '.build'

Downloading sources
Starting build of app.rambox.Rambox
Cache hit for rambox, skipping build
Cache hit for cleanup, skipping
Cache hit for finish, skipping
Everything cached, checking out from cache
Exporting app.rambox.Rambox to repo
Commit: d90965d689ea9cffd409e0b3aa69cdac7b5bd1f7827ab7f89493f4bccbd218b1
Metadata Total: 419
Metadata Written: 1
Content Total: 211
Content Written: 0
Content Bytes Written: 0 (0 bytes)
Exporting app.rambox.Rambox.Debug to repo
Commit: c0dda73559c1cb2be03ab842870e82d4da09501c4671c72d5b64da536e5e47f4
Metadata Total: 13
Metadata Written: 1
Content Total: 10
Content Written: 0
Content Bytes Written: 0 (0 bytes)
Installing app/app.rambox.Rambox/x86_64/master
F: No matching appdata for app.rambox.Rambox
Pruning cache`

With id set to the full ID this error doesn't appear!

Reply:
just asked in the flathub matrix chat. This makes absolutely no sense....